script/tool/test/build_examples_command_test.dart

```diff
@@ -59,7 +59,6 @@ void main() {
         ]),
       );
 
-      print(processRunner.recordedCalls);
```

Comment:
These changes are because I was copying from existing tests and noticed that they were spammy; this was presumably left from debugging the tests when they were being written.